index.html

```diff
@@ -57,93 +57,95 @@
-						<li class="settingsView active" title="Settings"><i class="fa fa-gear"></i></li>
-						<li class="txView" title="Transaction"><i class="fa fa-send"></i></li>
-						<li class="envView" title="Environment"><i class="fa fa-cube"></i></li>
-						<li class="publishView" title="Publish" ><i class="fa fa-cloud-upload"></i></li>
-						<li class="debugView" title="Debugger"><i class="fa fa-bug"></i></li>
-						<li class="verificationView" title="Formal Verification"><i class="fa fa-check"></i></li>
-						<li class="staticanalysisView" title="Static Analysis"><i class="fa fa-search"></i></li>
-						<li id="helpButton"><a href="https://solidity.readthedocs.org" target="_blank" title="Open Documentation" class="fa fa-question"></a></li>
+						<li class="settingsView active" title="Settings">Settings</li>
+						<li class="publishView" title="Publish" >Files</li>
+						<li class="envView" title="Environment">Contract</li>
+						<li class="debugView" title="Debugger">Debugger</li>
+						<li class="staticanalysisView" title="Static Analysis">Analysis</li>
+						<li id="helpButton"><a href="https://solidity.readthedocs.org" target="_blank" title="Open Documentation">Docs</a></li>
 					</ul>
 					<img id="solIcon" title="Solidity realtime compiler and runtime" src="assets/img/remix_logo_512x512.svg" alt="Solidity realtime compiler and runtime">
 				</div>
 				<div id="optionViews" class="settingsView">
 					<div id="txView">
-						<div class="crow">
-							<label for="txorigin"><select name="txorigin" id="txorigin"></select> Transaction origin</label>
-						</div>
-						<div class="crow">
-							<label for="gasLimit"><input type="number" id="gasLimit" value="3000000"> Transaction gas limit</label>
-						</div>
-						<div class="crow hide">
-							<label for="gasPrice"><input type="number" id="gasPrice" value="0"> Gas Price</label>
-						</div>
-						<div class="crow">
-							<label for="value"><input type="text" id="value" value="0"> Value (e.g. .7 ether or 5 wei, defaults to ether)</label>
-						</div>
 					</div>
 					<div id="settingsView">
-						<div class="version crow"><strong>Solidity version:</strong> <span id="version">(loading)</span></div>
-						<div class="crow">Change to: <select id="versionSelector"></select></div>
+						<div class="version crow"><strong>Current Solidity version:</strong> <span id="version">( Loading... )</span></div>
+						<div class="crow">Switch version: <select id="versionSelector"></select></div>
 						<div class="crow">
 							<label for="editorWrap"><input id="editorWrap" type="checkbox">Text Wrap</label>
 							<label for="optimize"><input id="optimize" type="checkbox">Enable Optimization</label>
 							<label for="autoCompile"><input id="autoCompile" type="checkbox" checked>Auto Compile</label>
-							<button id="compile" title="Compile source code"><i class="fa fa-cog"></i> Compile</button>
+							<button id="compile" title="Compile source code">Compile</button>
```

Comment:
@axic Hm. I had to rebase master branch so I stashed all my changes and then popped then back. 
So I joined all the commits in one -  **Rename and restructure tabs and content in righthand panel**. Do you think it will work or should I go back and redo all the cosmetic changes and make a new commit? 

I think we will be polishing the UI in the next weeks again, so I would rather skip this to be honest...